### ISSUE TITLE
Windows Instructions README updates 

### ISSUE THREAD

**Issue by ericoporto** [adventuregamestudio/ags#1085]
The Windows Instructions in [`README.md`](https://github.com/adventuregamestudio/ags/tree/master/Windows/README.md) in the [`Windows/`](https://github.com/adventuregamestudio/ags/tree/master/Windows) directory is incomplete.

It misses information on the following topics:

- [x] the inno installer and how it's built;
- [x] the usage of project properties or environment variables in Windows to get the project building
- ~~suggest where to place the libraries in the system~~
- [x] remove mentions on how to make the old manual (I think they are not needed anymore)
- [x] installation of [Windows 8.1 SDK](https://go.microsoft.com/fwlink/p/?LinkId=323507) from [SDK Archive](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive/). It used to be possible to install from VS itself but it's not anymore on the most recent VS 2019.
- [x] not sure if the windows patch is still needed for allegro, is allegro for Windows AGS on 4.4.2 ?
- ~~simple how to set AGSEditor as Startup project by right clicking on it in Solution Explorer (prevents "A project with an Output Type of Class Library cannot be started directly", because startup project by default is `AGS.Controls`);~~
- [x] You can't actually use the built editor from scratch because it comes without any templates;

Two small suggestions, not needed but nice to have:
- [x] reorder things so it's easier to get prebuilt libs+irrKlang refferences+env variable+Editor full solution running;
- [x] erh, maybe add the hack for debugging AGS with Allegro on Windows without suttering VS is a good one (not needed for SDL2);

There may be other things. I can do the PR with updated instructions just thought about first opening an issue to see if there is more things to mention or if I got something wrong.

**Comment by ivan-mogilko** [comment]
Added paragraph for building installer: 10e26bbf2df2155169fe9dc386cf07ce503393b8

**Comment by ericoporto** [comment]
I rearranged above as a list of things that can be ticked as done. I reread it and I think the mention of the patch is gone so I ticked it out too. Feel free to mark/remove things.

**Comment by ivan-mogilko** [comment]
Is the old manual building still mentioned somewhere? I don't see it, but maybe I'm missing something.

**Comment by ericoporto** [comment]
Nope, it's not, last remains was slashed in https://github.com/adventuregamestudio/ags/commit/4b51d35ec9030a3ca0e8181b512dff6e1a49d7ea

**Comment by persn** [comment]
> simple how to set AGSEditor as Startup project by right clicking on it in Solution Explorer (prevents "A project with an Output Type of Class Library cannot be started directly", because startup project by default is AGS.Controls);

We can make this set automatically to AGSEditor by twiddling the order of the appearance in the solution file. It's not obvious behavior in the solution format but it's possible to do. It should be fast and easy to do so I can take a look at it, and then we don't need to document it

**Comment by morganwillcock** [comment]
> Nope, it's not, last remains was slashed in [4b51d35](https://github.com/adventuregamestudio/ags/commit/4b51d35ec9030a3ca0e8181b512dff6e1a49d7ea)

The current instructions don't seem quite right as there is no reason that the Powershell script needs admin rights. Possibly it would be blocked by the Powershell execution policy, but that is a separate problem, and it is possible to bypass a restrictive policy by invoking the script in a particular way.

I'll look at changing the instructions.

**Comment by ivan-mogilko** [comment]
I also realized that the prebuilt libs archive is outdated, it corresponds to AGS 3.4.0 probably; I'll have to reupload it.

**Comment by ivan-mogilko** [comment]
Pushed another change: 83cc2c620b37f1151d38dcee69eecc8e8f313be0

* removed mention of OpenGL extension headers, as this should be covered by glad headers in our repository.
* added note about Windows 8.1 SDK for MSVS 2019.
* explained which enviroment variables to setup and how (link to msvs documentation).
* mention which projects are creating final output (executable), just in case.
* link to newer prebuilt libraries

**Comment by ivan-mogilko** [comment]
Probably last commit d2d2a02f5bdaea779d493aca6d6fae26714e15c4. I don't know what else to mention there.
Tbh, I don't know how to "suggest where to place the libraries in the system", and why does it matter.

**Comment by ericoporto** [comment]
OK, I marked things there on the top post accordingly.

**Comment by ivan-mogilko** [comment]
Seems like resolved.
